packages/bot-web-ui/src/stores/save-modal-store.ts

```diff
@@ -58,48 +65,63 @@ export default class SaveModalStore implements ISaveModalStore {
         return errors;
     };
 
-    addStrategyToWorkspace = async (
+    async addStrategyToWorkspace(
```

Comment:
any specific reason for removing arrow function and while calling await was removed. not a comment, just interested to know

Reply:
will revert back had changed to check something @vinu-deriv